server/cli/launch.py

```diff
@@ -342,32 +342,32 @@ def launch(
     else:
         port = find_available_port(host, DEFAULT_SERVER_PORT)
 
-    if not experimental_annotations:
-        if experimental_annotations_file is not None:
-            click.echo("Warning: --experimental-annotations-file ignored as --annotations not enabled.")
-        if experimental_annotations_output_dir is not None:
-            click.echo("Warning: --experimental-annotations-output-dir ignored as --annotations not enabled.")
-        if experimental_annotations_ontology:
-            click.echo("Warning: --experimental-annotations-ontology ignored as --annotations not enabled.")
-        if experimental_annotations_ontology_obo is not None:
-            click.echo("Warning: --experimental-annotations-ontology-obo ignored as --annotations not enabled.")
+    if disable_annotations:
+        if annotations_input_file is not None:
+            click.echo("Warning: --annotations-input-file ignored as annotations are disabled.")
```

Comment:
I think these should be hard errors, not warnings, as they imply a misconfiguration or other serious misunderstanding.  Warnings are easily missed in all the startup output.